src/components/publiq-ui/Modal/ContentModal.js

```diff
@@ -53,11 +54,13 @@ ContentModal.propTypes = {
   onShow: PropTypes.func,
   onClose: PropTypes.func,
   children: PropTypes.node,
+  size: PropTypes.string,
 };
 
 ContentModal.defaultProps = {
   visible: false,
   title: '',
+  size: 'xl',
```

Comment:
Is it possible to use the `ModalSizes.XL` constant here instead?

Reply:
Because the ModalSize lives in Another component `Modal`, It would throw error's that the constant was used before it was defined